packages/@atjson/renderer-react/package.json

```diff
@@ -16,10 +16,13 @@
     "access": "public"
   },
   "dependencies": {
-    "@atjson/renderer-hir": "file:../renderer-hir",
-    "react": "^16.2.0"
+    "@atjson/renderer-hir": "file:../renderer-hir"
+  },
+  "peerDependencies": {
+    "react": "*"
```

Comment:
I would consider using a major version pattern here so we handle any breaking changes in future React versions properly

Reply:
e.g. `16.x.x`

Reply:
Ok! I'm not sure if we'll encounter any, but it's probably safe to do (the only API we rely on is `React.createElement` in this)